src/napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -45,8 +45,8 @@
 
 def _show_message(widget):
     message = trans._(
-        'When installing/uninstalling npe2 plugins, '
-        'you must restart napari for UI changes to take effect.'
+        'Restart napari to activate changes. This is required when'
+        'installing/uninstalling npe2 plugins for the changes to take effect.'
```

Comment:
```suggestion
        'Restart napari after installing/uninstalling plugins'
```
What about even simpler?
Ideally this would appear when closing the manager but for now it's fine here.

Reply:
I like simple.